src/Altinn.Profile/Controllers/UsersController.cs

```diff
@@ -15,7 +16,7 @@ namespace Altinn.Profile.Controllers
     /// <summary>
     /// The users controller
     /// </summary>
-    [Authorize]
+    /// [Authorize]
```

Comment:
obs, this should be commented back in. Surprised all tests run green when this is commented out. Also something for us to follow up on.

Reply:
Fixed and added two tests to identify if this happens again test that fails when no authorization is performed